blog/docs/releases/announcing-knative-v1-10-release.md

```diff
@@ -0,0 +1,152 @@
+**Authors: [Paul Schweigert](https://github.com/psschwei) (IBM), [Evan Anderson](https://github.com/evankanderson) (VMware)**
```

Comment:
you probably want to add yourself and co-release leads here instead of me and Evan :smile: